content/docs/command-reference/commit.md

```diff
@@ -62,26 +64,26 @@ There are many cases where the last step is not desirable (for example rapid
 iterations on an experiment). The `--no-commit` option prevents the last step
 from occurring (on the commands where it's available), saving time and space by
 not storing unwanted <abbr>data artifacts</abbr>. The file hash is still
-computed and added to the DVC-file, but the actual data file is not saved in the
-cache. This is where the `dvc commit` command comes into play. It performs that
-last step (saving the data in cache).
+computed and added to the `dvc.lock` or `.dvc` file, but the actual data file is
+not saved in the cache. This is where the `dvc commit` command comes into play.
+It performs that last step (saving the data in cache).
 
 Note that it's best to avoid the last two scenarios. They essentially
-force-update the [DVC-files](/doc/user-guide/dvc-files-and-directories) and save
-data to cache. They are still useful, but keep in mind that DVC can't guarantee
-reproducibility in those cases.
+force-update the `dvc.lock` or `.dvc` files and save data to cache. They are
+still useful, but keep in mind that DVC can't guarantee reproducibility in those
+cases.
 
 ## Options
 
 - `-d`, `--with-deps` - determines files to commit by tracking dependencies to
-  the target DVC-files (stages). If no `targets` are provided, this option is
-  ignored. By traversing all stage dependencies, DVC searches backward from the
-  target stages in the corresponding pipelines. This means DVC will not commit
-  files referenced in later stages than the `targets`.
+  the target stages or `.dvc` files. If no `targets` are provided, this option
+  is ignored. By traversing all stage dependencies, DVC searches backward from
+  the target stages in the corresponding pipelines. This means DVC will not
+  commit files referenced in later stages than the `targets`.
```

Comment:
OK for now but

- [ ] We need to update this option desc. in several cmd refs.

(out of scope for this PR)

Reply:
Here it should only be stages? If I am not wrong then `.dvc` doesn't have a dependency.

Reply:
The file tracked by `.dvc` is its dependency. So the text is correct.

Reply:
That part is correct but there's no mention of dvc.yaml and the text could probably be simplified a little. It's OK for now.